docs/self-hosted/quickstart.md

```diff
@@ -1,158 +1,64 @@
 
-You can use the following configuration to get started using Docker Compose. Make sure to change all sensitive values
-like `KEY`, `SECRET`, `ADMIN_PASSWORD`, _etc._
+As soon as there are new releases of Directus, we publish them on [Docker Hub](https://hub.docker.com/r/directus/directus).
 
-```yaml
+:::
+
+## Create a Docker Compose File
+
+Create a new empty directory, and open it in a text editor. Create a `docker-compose.yml` file and paste the following:
+
+```yml
 version: '3'
 services:
   directus:
     image: directus/directus:latest
     ports:
       - 8055:8055
     volumes:
-      - ./uploads:/directus/uploads
       - ./database:/directus/database
+      - ./uploads:/directus/uploads
     environment:
-      KEY: '255d861b-5ea1-5996-9aa3-922530ec40b1'
-      SECRET: '6116487b-cda1-52c2-b5b5-c8022c45e263'
-
-      DB_CLIENT: 'sqlite3'
-      DB_FILENAME: './database/data.db'
-
+      KEY: 'replace-with-random-value'
+      SECRET: 'replace-with-random-value'
       ADMIN_EMAIL: 'admin@example.com'
       ADMIN_PASSWORD: 'd1r3ctu5'
+      WEBSOCKETS_ENABLED: true
 ```
 
-Save this in your project as a file named `docker-compose.yml` and run:
-
-```
-docker-compose up -d
-```
-
-::: tip More Info on Docker
-
-To learn more, visit the [Docker Guide](/self-hosted/docker-guide).
-
-:::
-
-## 2. Login to App
-
-With the server running, you're now able to login to your new Directus project and start using it.
-
-Our start command stated that the server started at port `8055`, which means we can navigate to `http://localhost:8055`
-to open Directus in the browser.
-
-Login using the admin credentials you configured during the installation in Step 1.
-
-## 3. Create a Collection
-
-Once logged in, you're greeted with the option to create your first Collection:
-
-![Directus Empty State](https://cdn.directus.io/docs/v9/getting-started/quickstart/quickstart-20220217A/empty-state-20220217A.webp)
-
-Follow the prompts and create a Collection. For the sake of this demo, we'll be calling ours `articles`, but feel free
-to make it your own!
-
-::: tip More Info on Collections
-
-To learn more, see our documentation [Collections](/app/content/collections).
-
-:::
+Save the file. Let's step through it:
 
-## 4. Create a Field
+- This file defines a single Docker container that will use the latest version of the `directus/directus` image.
+- The `ports` list maps internal port `8055` is made available to our machine using the same port number, meaning we can access it from our computer's browser.
+- The`volumes` section maps internal `directus/database` and `directus/uploads` to our local file system alongside the `docker-compose.yml` - meaning data is backed up outside of Docker containers.
+- The `environment` section contains any [configuration variables](/self-hosted/config-options.html) we wish to set.
+  - `KEY` and `SECRET` are required and should be random values.
```

Comment:
Not sure whether there are certain requirements (length, ...) for these values 🤔 cc @br41nslug

Reply:
I have previously used 'key' and 'secret' and it was fine, so I'd assume not.

Reply:
Ah yes mentioning what it's for will help explain why it needs to be random and long. I'll do this

Reply:
Awesome, thanks ❤️